### PR TITLE
Add date support in calculation fields

### PR DESCRIPTION
**Pull request by pboissonneault**
Hi, I just added date support in the calculation fields. When a date is specified, convert it to days, so we can compare dates and have days between them.

### CODE REVIEW

File: assets/js/conditionals.js [hunk 1]
```diff
@@ -124,10 +124,18 @@ var calders_forms_check_conditions, calders_forms_init_conditions;
 					}
 				}				
 				// add result in
-				istrue = true;
+				//PBPB : Change AND / OR
```
Comment (michaelbeil) [inline assets/js/conditionals.js]: Was there a reason you left these commented out lines in?

**Comment by michaelbeil** [comment]
It appears like you have renamed Caldera Forms to Caldera Forms - UdeS - FLSH. This must be a change you made for your own usage?

**Comment by pboissonneault** [comment]
I just wanted to make sure my plugin doesn't update and to have OR instead
of AND on conditions, I'm not much familiar with Github, tThe pull request
I sent was the current-stable on January 25, I've push other little changes
since, that but you don't want them.

**Comment by pboissonneault** [comment]
I reverted it and created a new branch. The changes in current-stable might
interest you

**Comment by Shelob9** [comment]
Hey @Desertsnowman this can't be merged -- wrong branch, too many differences -- but, it looks neat, can you see if you can pull some of this in?

**Comment by Desertsnowman** [comment]
@pboissonneault - this is great, so sorry I missed it earier. Could you possible redo this on the `current-dev` branch and do another pull request?

**Pull request closed without merging**